design2.css

```diff
@@ -0,0 +1,136 @@
+body {
+	background-color: #008800;
+	font-family: Verdana, Arial, Helvetica, sans-serif;
+	text-align: center;
+}
+* {

```

Comment:
Good usage of global pattern but it is generally good practice to avoid it. So we can prevent some of the pitfalls when scaling styles.